Assets/MixedRealityToolkit/Utilities/ProximityLight.cs

```diff
@@ -124,7 +124,7 @@ public Color OuterColor
             [Tooltip("The color of the ProximityLight gradient at the outer (RGB) and (A) is gradient extent.")]
             [SerializeField]
             [ColorUsageAttribute(true, true)]
-            private Color outerColor = new Color((70.0f * 5.0f) / 255.0f, (0.0f * 5.0f) / 255.0f, (191.0f * 5.0f) / 255.0f, 255.0f / 255.0f);
+            private Color outerColor = new Color((82.0f * 3.0f) / 255.0f, (31.0f * 3.0f) / 255.0f, (191.0f * 3.0f) / 255.0f, 255.0f / 255.0f);
```

Comment:
Why times 3 instead of just putting like 246 / 255?

Reply:
There's some other weird stuff here like 255.0f/255.0f (could just be 1.0f). I guess maybe the goal was consistency with the rest of the line but it does feel like there's just noise here.

Reply:
These are great questions! I did that to match the values found in the default color picker for HDR colors. (In recent versions of Unity they have also exposed floating point color values.) But, by default an HDR color value is displayed in Unity as 3 unsigned byte values and a floating point scaler.